sqlmesh/core/snapshot/cache.py

```diff
@@ -46,15 +49,37 @@ def get_or_load(
         if snapshot_ids_to_load:
             loaded_snapshots = loader(snapshot_ids_to_load)
             for snapshot in loaded_snapshots:
-                self._update_node_hash_cache(snapshot)
-                self.put(snapshot)
```

Comment:
If you removed this, how the snapshots will be cached after being fetched?

Reply:
doesn't get it put on line 77?